docs/4.0/trustedclusters.md

```diff
@@ -240,7 +268,7 @@ Now an admin from the main cluster can now see and access the "east" cluster:
 
 ```bsh
 # login into the main cluster:
-$ tsh --proxy=proxy.main login admin
+$ tsh --proxy=main.example.com login admin
```

Comment:
If we're using `main.example.com` here and elsewhere in the docs and `<main-proxy-address>` above it's probably a little confusing; I'd stick with `main.example.com` throughout.